### ISSUE TITLE
Please add C coding conventions

### ISSUE THREAD

**Issue by esoule** [outsideris/popularconvention#47]
Please add C coding conventions

I wrote a short guide based on indent man page, Wikipedia and this project.

https://gist.github.com/esoule/7931356

**Comment by nowox** [comment]
Yeah adding C/C++ could be a very nice as the debates with those languages is usually very strong
